refinery/core/models.py

```diff
@@ -181,7 +182,7 @@ def register_handler(request, sender, user, **kwargs):
 
 
 # check if user has a catch all project and create one if not
-def create_catch_all_project(sender, user, request, **kwargs):
+def create_catch_all_project(user, request):
```

Comment:
Not for this pull request, but did you get a chance to see how difficult it'd be to disentangle project and remove it completely?